powershell_shortcut-feedstock/recipe/menu-windows-ps.json

```diff
@@ -0,0 +1,12 @@
+{
+    "menu_name": "Anaconda${PY_VER} ${PLATFORM}",
+    "menu_items":
+    [
+        {
+            "name": "Anaconda Powershell Prompt",
+            "system": "%windir%\\System32\\WindowsPowerShell\\v1.0\\powershell.exe",
+            "scriptarguments": ["/NoExit", "/Command {conda activate", "${PREFIX} }"],
```

Comment:
I might be wrong here, but doesn't this assume that ``conda init`` has been run?  What happens if it hasn't been run?  Is there a way to change this command to run the script instead, so that this shortcut will work regardless of whether ``conda init`` has been run?